{
 "archived": false,
 "branch": "main",
 "conda-forge.yml": {
  "conda_build": {
   "pkg_format": "2"
  },
  "conda_forge_output_validation": true,
  "github": {
   "branch_name": "main",
   "tooling_branch_name": "main"
  }
 },
 "feedstock_name": "django-denied",
 "hash_type": "sha256",
 "linux_64_meta_yaml": {
  "about": {
   "home": "https://github.com/mblayman/django-denied",
   "license": "MIT",
   "license_file": [
    "LICENSE"
   ],
   "summary": "An authorization system based exclusively on allow lists"
  },
  "build": {
   "noarch": "python",
   "number": "1",
   "script": "PYTHON -m pip install . -vv"
  },
  "extra": {
   "recipe-maintainers": [
    "thewchan"
   ]
  },
  "package": {
   "name": "django-denied",
   "version": "1.3"
  },
  "requirements": {
   "host": [
    "pip",
    "python 3.9",
    "setuptools"
   ],
   "run": [
    "django >=3.2",
    "python >=3.9"
   ]
  },
  "schema_version": 0,
  "source": {
   "sha256": "130da88b74984c36fd8deef44e5cec7b759828c514f1fffcb541a12b1c9f96fa",
   "url": "https://pypi.org/packages/source/d/django-denied/django-denied-1.3.tar.gz"
  },
  "test": {
   "commands": [
    "pip check"
   ],
   "imports": [
    "denied"
   ],
   "requires": [
    "pip",
    "python 3.9"
   ]
  }
 },
 "linux_64_requirements": {
  "build": {
   "__set__": true,
   "elements": []
  },
  "host": {
   "__set__": true,
   "elements": [
    "pip",
    "python",
    "setuptools"
   ]
  },
  "run": {
   "__set__": true,
   "elements": [
    "django",
    "python"
   ]
  },
  "test": {
   "__set__": true,
   "elements": [
    "pip",
    "python"
   ]
  }
 },
 "meta_yaml": {
  "about": {
   "home": "https://github.com/mblayman/django-denied",
   "license": "MIT",
   "license_file": [
    "LICENSE"
   ],
   "summary": "An authorization system based exclusively on allow lists"
  },
  "build": {
   "noarch": "python",
   "number": "1",
   "script": "PYTHON -m pip install . -vv"
  },
  "extra": {
   "recipe-maintainers": [
    "thewchan"
   ]
  },
  "package": {
   "name": "django-denied",
   "version": "1.3"
  },
  "requirements": {
   "host": [
    "pip",
    "python 3.9",
    "setuptools"
   ],
   "run": [
    "django >=3.2",
    "python >=3.9"
   ]
  },
  "schema_version": 0,
  "source": {
   "sha256": "130da88b74984c36fd8deef44e5cec7b759828c514f1fffcb541a12b1c9f96fa",
   "url": "https://pypi.org/packages/source/d/django-denied/django-denied-1.3.tar.gz"
  },
  "test": {
   "commands": [
    "pip check"
   ],
   "imports": [
    "denied"
   ],
   "requires": [
    "pip",
    "python 3.9"
   ]
  }
 },
 "name": "django-denied",
 "outputs_names": {
  "__set__": true,
  "elements": [
   "django-denied"
  ]
 },
 "parsing_error": "feedstock parsing error: Error running 'conda-forge-tick-container parse-feedstock --existing-feedstock-node-attrs - --log-level info --mark-not-archived' in container - error UnboundLocalError raised:\n\"cannot access local variable 'r' where it is not associated with a value\"\nTraceback (most recent call last):\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/make_graph.py\", line 102, in try_load_feedstock\n    data = load_feedstock(name, attrs.data, mark_not_archived=mark_not_archived)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/feedstock_parser.py\", line 705, in load_feedstock\n    return load_feedstock_containerized(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/feedstock_parser.py\", line 657, in load_feedstock_containerized\n    data = run_container_operation(\n           ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.11/site-packages/conda_forge_feedstock_ops/container_utils.py\", line 221, in run_container_operation\n    raise ContainerRuntimeError(\nconda_forge_feedstock_ops.container_utils.ContainerRuntimeError: Error running 'conda-forge-tick-container parse-feedstock --existing-feedstock-node-attrs - --log-level info --mark-not-archived' in container - error UnboundLocalError raised:\n\"cannot access local variable 'r' where it is not associated with a value\"\n",
 "platforms": [
  "linux_64"
 ],
 "pr_info": {
  "__lazy_json__": "pr_info/django-denied.json"
 },
 "raw_meta_yaml": "{% set name = \"django-denied\" %}\n{% set version = \"1.3\" %}\n\npackage:\n  name: {{ name|lower }}\n  version: {{ version }}\n\nsource:\n  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/django-denied-{{ version }}.tar.gz\n  sha256: 130da88b74984c36fd8deef44e5cec7b759828c514f1fffcb541a12b1c9f96fa\n\nbuild:\n  noarch: python\n  script: {{ PYTHON }} -m pip install . -vv\n  number: 1\n\nrequirements:\n  host:\n    - pip\n    - python {{ python_min }}\n    - setuptools\n  run:\n    - django >=3.2\n    - python >={{ python_min }}\n\ntest:\n  imports:\n    - denied\n  commands:\n    - pip check\n  requires:\n    - pip\n    - python {{ python_min }}\n\nabout:\n  home: https://github.com/mblayman/django-denied\n  summary: An authorization system based exclusively on allow lists\n  license: MIT\n  license_file:\n    - LICENSE\n\nextra:\n  recipe-maintainers:\n    - thewchan\n",
 "req": {
  "__set__": true,
  "elements": [
   "django",
   "pip",
   "python",
   "setuptools"
  ]
 },
 "requirements": {
  "build": {
   "__set__": true,
   "elements": []
  },
  "host": {
   "__set__": true,
   "elements": [
    "pip",
    "python",
    "setuptools"
   ]
  },
  "run": {
   "__set__": true,
   "elements": [
    "django",
    "python"
   ]
  },
  "test": {
   "__set__": true,
   "elements": [
    "pip",
    "python"
   ]
  }
 },
 "strong_exports": false,
 "total_requirements": {
  "build": {
   "__set__": true,
   "elements": []
  },
  "host": {
   "__set__": true,
   "elements": [
    "pip",
    "python 3.9",
    "setuptools"
   ]
  },
  "run": {
   "__set__": true,
   "elements": [
    "django >=3.2",
    "python >=3.9"
   ]
  },
  "test": {
   "__set__": true,
   "elements": [
    "pip",
    "python 3.9"
   ]
  }
 },
 "url": "https://pypi.org/packages/source/d/django-denied/django-denied-1.3.tar.gz",
 "version": "1.3",
 "version_pr_info": {
  "__lazy_json__": "version_pr_info/django-denied.json"
 }
}